src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -159,110 +163,9 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
     }
   }
 
-  private renderStyle(): TemplateResult {
-    return html`
-      <style>
-        :host {
-          display: block;
-        }
-
-        ha-card {
-          position: relative;
-          overflow: hidden;
-          --name-font-size: 1.2rem;
-          --brightness-font-size: 1.2rem;
-          --rail-border-color: transparent;
-        }
-
-        #tooltip {
-          position: absolute;
-          top: 0;
-          left: 0;
-          right: 0;
-          height: 100%;
-          text-align: center;
-        }
-
-        #light {
-          margin: auto;
-          padding-top: 0;
-          padding-bottom: 32px;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          height: 160px;
-          width: 160px;
-        }
-        #light round-slider {
-          margin: 0 auto;
-          display: inline-block;
-          --round-slider-path-color: var(--disabled-text-color);
-          --round-slider-bar-color: var(--primary-color);
-          z-index: 20;
-        }
-
-        .light-icon {
-          position: absolute;
-          margin: 0 auto;
-          width: 76px;
-          height: 76px;
-          color: var(--paper-item-icon-color, #44739e);
-          cursor: pointer;
-          z-index: 20;
-        }
-
-        .light-icon[data-state="on"] {
-          color: var(--paper-item-icon-active-color, #fdd835);
-        }
-
-        .light-icon[data-state="unavailable"] {
-          color: var(--state-icon-unavailable-color);
-        }
-
-        .name {
-          position: absolute;
-          font-size: var(--name-font-size);
-          bottom: 16px;
-          box-sizing: border-box;
-          text-align: center;
-          width: 100%;
-          padding: 0 16px;
-        }
-
-        .brightness {
-          font-size: var(--brightness-font-size);
-          position: absolute;
-          margin: 0 auto;
-          top: 135px;
-          left: 50%;
-          transform: translate(-50%);
-          opacity: 0;
-          transition: opacity 0.5s ease-in-out;
-          -moz-transition: opacity 0.5s ease-in-out;
-          -webkit-transition: opacity 0.5s ease-in-out;
-          cursor: pointer;
-          pointer-events: none;
-        }
-
-        .show_brightness {
-          opacity: 1;
-        }
-
-        .more-info {
-          position: absolute;
-          cursor: pointer;
-          top: 0;
-          right: 0;
-          z-index: 25;
-          color: var(--secondary-text-color);
-        }
-      </style>
-    `;
-  }
-
   private _dragEvent(e: any): void {
     this.shadowRoot!.querySelector(".brightness")!.innerHTML =
-      e.detail.value + "%";
+      "&nbsp;" + e.detail.value + " %";
```

Comment:
Why the `&nbsp;` and please use a template literal

Reply:
`&nbsp`s are for alignment.
I don't want `50 %` to be perfectly centered, but shifted slightly to the right for balance.
I suppose the same thing could be done with an `em`-based offset, but this was simpler.

Reply:
Can't you use padding?

Reply:
That works.